third_party/gems/bundle-env.sh

```diff
@@ -0,0 +1,34 @@
+#!/bin/bash
+
```

Comment:
Can we add a comment explaining what this script is supposed to take & do?

Reply:
I've added a comment.